modules/policy_document/variables.tf

```
@@ -4,12 +4,8 @@ variable "template" {
 }
 
 variable "template_paths" {
-  description = "List of directory paths to search for the `template` policy document. Max of 10 paths. Policy statements are merged by SID in list order"
+  description = "List of directory paths to search for the `template` policy document. Policy statements are merged by SID in list order. If the `template` does not exist at a given template_path, that path is ignored"
```

Maybe more clear to mention empty policy is inserted?

```suggestion
  description = "List of directory paths to search for the `template` policy document. Policy statements are merged by SID in list order. If the `template` does not exist at a given template_path, an empty policy is inserted"
```

---

That probably makes sense, considering if the template doesn't exist at _any_ template_paths, then the result is an empty policy. Would helps explain _why_ that occurs... 😂

---

Done, re-running tests...